src/SearchResults.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import Moment from "moment";
+import "./App.css";
+
+export default function SearchResults({ results, handleProfile }) {
+  // let bookings = [
+  //   {
+  //     id: 1,
+  //     title: "Mr",
+  //     firstName: "John",
+  //     surname: "Doe",
+  //     email: "johndoe@doe.com",
+  //     roomId: 2,
+  //     checkInDate: "2017-11-21",
+  //     checkOutDate: "2017-11-23"
+  //   },
+  //   {
+  //     id: 2,
+  //     title: "Doctor",
+  //     firstName: "Sadia",
+  //     surname: "Begum",
+  //     email: "begum_sadia@sadia.org",
+  //     roomId: 1,
+  //     checkInDate: "2018-02-15",
+  //     checkOutDate: "2018-02-28"
+  //   }
+  // ];
+  let bookings = results;
+
+  const [bookingColor, setbookingColor] = useState({});
+
+  const changeColor = bookingId => {
+    return setbookingColor(bookingColor => {
+      if (bookingColor[bookingId] === "yellow") {
+        let updated = { ...bookingColor };
+        updated[bookingId] = "white";
+        return updated;
+      } else {
+        let updated = { ...bookingColor };
+        updated[bookingId] = "yellow";
+        return updated;
+      }
+    });
+  };
```

Comment:
good job on getting each row to highlight on click. A simpler way to achieve this could be to create a component for each row, each row can then have its own state, which can be toggled on click (for example a highlighted state value could be toggled between true and false, with a style/className being set depending on whether it is true or false)